src/App.js

```diff
@@ -1,35 +1,24 @@
-import { DataQuery } from '@dhis2/app-runtime'
-import i18n from '@dhis2/d2-i18n'
+import { CssVariables } from '@dhis2/ui'
 import React from 'react'
-import classes from './App.module.css'
+import './App.css'
+import { Layout, Header, Main, Sidebar, Footer } from './layout/index.js'
```

Comment:
I think it'd be better if these names were aligned with the design docs (regardless of internally used elements):
* `Header -> SelectorToolbar`
* `Main -> DataWorkspace / Workspace`
* `Sidebar -> RightPanel`
* `Footer -> BottomBar`

Reply:
I started out with that, but tbh I found it rather cumbersome. It introduces all kinds of awkward long names, and doesn't immediately make sense when working in the app. I can imagine these names would maybe make sense for the components rendered _in_ the layout, but coupling the layout slots to names from the design seems a bit excessive to me. Doesn't really make the code easier to read or work on for me, the opposite in fact.

Reply:
But that's my opinion of course, so let me know if you think it's not a valid line of reasoning.